src/lib/ng-v5/entry-point/write-package.transform.ts

```diff
@@ -17,6 +17,7 @@ export const writePackageTransform: Transform = transformFromPromise(async graph
 
   // 5. COPY SOURCE FILES TO DESTINATION
   log.info('Copying staged files');
+  copyFiles(`${path.dirname(ngEntryPoint.entryFilePath)}/**/*.d.ts`, entryPoint.data.outDir);
```

Comment:
Omg, I though the `writeFile` callback was capturing the triple-slash referenced typings. It isn't.